include/hermes/Support/Conversions.h

```diff
@@ -88,7 +88,7 @@ inline uint32_t truncateToUInt32(double d) {
 /// "A property name P (in the form of a String value) is an array index if and
 /// only if ToString(ToUint32(P)) is equal to P and ToUint32(P) is not equal to
 /// 2**32−1."
-/// IT must meet the requirements of InputIterator and its 'value_type' must be
+/// It must meet the requirements of InputIterator and its 'value_type' must be
```

Comment:
This is supposed to be IT

Reply:
Oops. I did not realize that IT refers to a template parameter. Reverted.